interview-questions.md

```diff
@@ -8,43 +8,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 3. Name three rails generator commands. What is created by each?
-
+  
   Your answer:
+  Rails generate controller, model, and migrate. 
 
   Researched answer:
-
-
+  Three rails generator commands are rails generate controller, model, and migrate. Rails generate commands create templates. When running rails generate controller, the controller is created. The controller can be called whatever the app you are creating is intended for. If creating a blog post app, the controller could be named blog_posts. The generate controller uses a boiler plate and creates all the files associated with a controller.
+  The generate command for model takes the name of the model we intend to create, the name of each column, and the data type of each column as arguments. If creating a blog post, the syntax could be $ rails generate model Blog title:string content:string.
+  Rails generate migrate allows you to make changes to a database. As an example, some of the information that could be changed is the column name, data type, or even adding or removing columns. A migration can also be ran if the developer forgets to include a foreign key when generating a model. Always remember to run $ rails db:migrate in order to execute the changes.
 
 4. Consider the Rails routes below. What is the name of the controller method that would be called by each route? What action would each of the controller methods perform?
 
-action: "GET"    location: /students          
+action: "GET"    location: /students
+  Index. This would list all students.          
 
 action: "POST"   location: /students       
+  Create. This would create a new student in the database then redirect somewhere.
 
 action: "GET"    location: /students/new
+  New. Show a new student form.
 
 action: "GET"    location: /students/2  
+  Show. Show info about specific student with id 2.
 
 action: "GET"    location: /students/2/edit    
+  Edit. Show edit form for student with id 2.
 
 action: "PATCH"  location: /students/2      
+  Update. Update particular student with id 2 then redirect somewhere.
 
 action: "DELETE" location: /students/2      
-
-
+  Destroy. Delete particular student with id 2 then redirect somewhere.
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
```

Comment:
This question was to solely concentrate on the user experience with the web oage